website/src/pages/pricing.tsx

```diff
@@ -255,7 +256,14 @@ export default ((props: any) => (
                         </p>
                     </div>
                     <div className="col-md-6 pt-3 align-self-center text-center">
-                        <Link className="btn btn-primary mx-2 mb-3" to="../#get-started" title="Try Sourcegraph now">
+                        <Link
+                            className="btn btn-primary mx-2 mb-3"
+                            data-button-style={buttonStyle.primary}
+                            data-button-location={buttonLocation.trySourcegraph}
+                            data-button-type="cta"
+                            to="../#get-started"
```

Comment:
Is `../#` a typo? I think it should be `/get-started`

Reply:
Ohh yeah that's a typo! Thanks for catching.